README.md

```diff
@@ -20,3 +22,23 @@ The MiPaPo follows the [Semantic Versioning 2.0.0](https://semver.org/).
 
 Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format. 
 
+# How to install
+## Settings
+To setup the application you need a file named `.psh.yaml.override` in your **root directory**.
+After you have that go inside the `.psh.yaml`, crap all settings you wanna change and paste them inside the `.psh.yaml.override`.
```

Comment:
crap as in taking a shit? not the right word.

Reply:
copy**

Reply:
Done --> https://github.com/xPand4B/MiPa-Pool/pull/16/commits/d80c93164dce6e506cddd9ab2d44ee04ebb4f094